### PR TITLE
Documenting Rest Client TLS Registry usage to disable SSL

### LINKED ISSUE #45193: Rest Client, Document disabling ssl using tls registry
### Description

Take care of adjusted with TLS Registry instructions to make Disabling SSL of specific Rest Client (Battle tested, it works).

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by lasteris**
- Fixes: #45193

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Thanks!

File: docs/src/main/asciidoc/rest-client.adoc [hunk 1]
```diff
@@ -492,18 +492,33 @@ quarkus.rest-client.extensions-api.scope=jakarta.inject.Singleton
 Setting the base URL of the client is **mandatory**, however the REST Client supports per-invocation overrides of the base URL using the `@io.quarkus.rest.client.reactive.Url` annotation.
 ====
 
-=== Disabling Hostname Verification
+=== Trusting all certificates and Disabling SSL hostname verification
 
-To disable the SSL hostname verification for a specific REST client, add the following property to your configuration:
+[IMPORTANT]
```
Comment (geoand) [inline docs/src/main/asciidoc/rest-client.adoc:497]:
```suggestion
[WARNING]
```

**Comment by lasteris** [comment]
> Please squash the commits :)

yeah, i've just need a bit more time for this :DD

**Review by geoand — APPROVED** [review]
Thanks!

**Pull request merged**